macros/ui/source.pl

```diff
@@ -1,3 +1,5 @@
+# Note: this probably no longer works using CGI unless specifically set up.  Deprecate?
```

Comment:
As with all of these comments that you have added at the beginnings of files, remove it.  Also, note that this should not be deprecated.  It does take special set up (and always has), but it still has its uses and does still work.  Of course it should never be used in a problem for a course, and should not be used in problems added to the OPL.

Reply:
I was thinking depcrecating this because I didn't think that CGI scripts would currently work with most WW installations--maybe reconfigure mojolicious to do this for us. 

I added comments at the top of other lines--I meant to ask others about deprecating other macros.